tests/gold_tests/autest-site/trafficserver.test.ext

```diff
@@ -121,7 +121,7 @@ def MakeATSProcess(obj, name, command='traffic_server', select_ports=True):
     # create subdirectories that need to exist (but are empty)
     # log directory has to be created with correct permissions
     p.Setup.MakeDir(log_dir)  # log directory has to be created
-    p.Setup.Chown(log_dir, "nobody", "nobody", ignore=True)
+    p.Setup.Chown(log_dir, "nobody", "nogroup", ignore=True)
```

Comment:
I want to make sure we have the correct "value" for security being tested

@bryancall @SolidWallOfCode Is this a better value?

Reply:
on RHEL 7, the group name is nobody with gid 99. But on Fedora/Ubuntu, the corresponding group is nogroup with gid 65534. So I'm not sure what to put for the group field in chown. Calling chown twice to try both nobody and nogroup seems like a not-so-good solution.

Reply:
You can grep through /etc/passwd to see which builtin user is present.

grep "^nobody:" /etc/passwd

Alternatively, does our CI system create an "ats" user for the tests to use @zwoop?

Reply:
I would check what the default group is for nobody and use that.